samples/drivers/led_strip/boards/adafruit_metro_esp32s3_procpu.overlay

```diff
@@ -32,4 +32,4 @@
 		LED_COLOR_ID_RED
 		LED_COLOR_ID_BLUE>;
 	};
-};
\ No newline at end of file
+};
```

Comment:
this should be fixed in the original commit, not added in one then fixed in another

Reply:
Good catch. Should be fixed now. I'm honestly getting a little dizzy from all this rewriting of git history. Hopefully I didn't make any mistakes this time.